### ISSUE TITLE
cant deploy azure funciton with pyodbc

### ISSUE THREAD

**Issue by MariusFearn** [microsoft/vscode-azurefunctions#1314]
Repro steps:

1.
2.

Action: azureFunctions.deploy
Error type: SyntaxError
Error Message: Connection strings must be of the form "key1=value1;key2=value2".

Version: 0.17.1
OS: win32
Product: Visual Studio Code
Product Version: 1.34.0
Language: en

<details>
<summary>Call Stack</summary>

```
Object.module.exports.t.parseAndValidateKeys extension.bundle.js:164:169113
Function.module.exports.O.createFromConnectionString extension.bundle.js:332:106643
Function.module.exports.j.getStorageSettings extension.bundle.js:332:62358
new D extension.bundle.js:332:136345
Object.module.exports.t.createBlobService extension.bundle.js:164:151520
extension.bundle.js:332:7349extension.bundle.js:332:7349
o extension.bundle.js:332:6713
process._tickCallback next_tick.js:68:7
```

</details>

**Comment by EricJizbaMSFT** [comment]
@MariusFearn can you add repro steps to this issue?

**Comment by MariusFearn** [comment]
I am pushing a python function to Azure with odbc connection (pyodbc) using HttpTrigger

I got it to work in the end, the fix for me was simply to restart from scratch, I had tried to push several version in several ways, so I guess somehow I broke it along the way. My hunch is that this is somehow related to the requirment.txt and forcing func with --build-native-deps. I build the function first without this parameter, and tried to push, then with this parameter, and back and forth several times.

That said I also experienced issues posting to westeurope (here it was syncing issues), switch to westus and it this also worked in the end.
